<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartOptionsDialog</class>
 <widget class="QDialog" name="ChartOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Chart...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/chart-edit</normaloff>:/icons/chart-edit</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Chart Type</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QRadioButton" name="rbCtHumidity">
        <property name="text">
         <string>Humidity</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="rbCtPressure">
        <property name="text">
         <string>Pressure</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="rbCtTemperature">
        <property name="text">
         <string>Temperature</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="rbCtRainfall">
        <property name="text">
         <string>Rainfall</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Data Sets</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="0">
       <widget class="QCheckBox" name="cbWindChill">
        <property name="text">
         <string>Wind Chill</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="cbHumidity">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Humidity</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="cbIndoorHumidity">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Humidity (inside)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="cbDewPoint">
        <property name="text">
         <string>Dew Point</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="cbTemperature">
        <property name="text">
         <string>Temperature</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="cbApparentTemperature">
        <property name="text">
         <string>Apparent Temperature</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="cbIndoorTemperature">
        <property name="text">
         <string>Temperature (inside)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="cbPressure">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Pressure</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="cbRainfall">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Rainfall</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Period</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QRadioButton" name="rbTToday">
        <property name="text">
         <string>Today</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="rbTThisYear">
        <property name="text">
         <string>This year</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="rbTYesterday">
        <property name="text">
         <string>Yesterday</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="rbTThisWeek">
        <property name="text">
         <string>This week</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="rbTThisMonth">
        <property name="text">
         <string>This month</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QRadioButton" name="rbTCustom">
        <property name="text">
         <string>Custom:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDateTimeEdit" name="startTime">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="displayFormat">
         <string>d/M/yyyy h:mm:ss AP</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>to</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QDateTimeEdit" name="endTime">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="displayFormat">
         <string>d/M/yyyy h:mm:ss AP</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>rbCtTemperature</tabstop>
  <tabstop>rbCtHumidity</tabstop>
  <tabstop>rbCtPressure</tabstop>
  <tabstop>cbTemperature</tabstop>
  <tabstop>cbIndoorTemperature</tabstop>
  <tabstop>cbApparentTemperature</tabstop>
  <tabstop>cbWindChill</tabstop>
  <tabstop>cbDewPoint</tabstop>
  <tabstop>cbHumidity</tabstop>
  <tabstop>cbIndoorHumidity</tabstop>
  <tabstop>cbPressure</tabstop>
  <tabstop>rbTToday</tabstop>
  <tabstop>rbTYesterday</tabstop>
  <tabstop>rbTThisWeek</tabstop>
  <tabstop>rbTThisMonth</tabstop>
  <tabstop>rbTThisYear</tabstop>
  <tabstop>rbTCustom</tabstop>
  <tabstop>startTime</tabstop>
  <tabstop>endTime</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ChartOptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
